packages/block-library/src/cover/edit/inspector-controls.js

```diff
@@ -198,6 +201,7 @@ export default function CoverInspectorControls( {
 								focalPoint: undefined,
 								isRepeated: false,
 								alt: '',
+								poster: '',
```

Comment:
```suggestion
								poster: undefined,
```

I think this value should be `undefined`, otherwise the `poster` attribute with an empty value remains when resetting: 

<img width="352" height="106" alt="image" src="https://github.com/user-attachments/assets/d3ff1fab-dd3c-4cde-9ccf-7213328a5f29" />